init/main.go

```diff
@@ -0,0 +1,149 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"math"
+	"math/rand"
+	"net"
+	"net/http"
+	"strconv"
+
+	"k8s.io/kubernetes/pkg/api"
+	k8sRestCl "k8s.io/kubernetes/pkg/client/restclient"
+	k8sClient "k8s.io/kubernetes/pkg/client/unversioned"
+	"k8s.io/kubernetes/pkg/fields"
+	"k8s.io/kubernetes/pkg/labels"
+)
+
+const (
+	k8sHost         = "127.0.0.1"
+	k8sPort         = "8080"
+	port            = "8090"
+	joulesLabelName = "joules"
```

Comment:
I don't think it is necessary as this code is not very relevant for other people, other than maybe nerdalize who probably don't mind what the label is called